blockchain/blockdao.go

```diff
@@ -88,36 +88,41 @@ func (dao *blockDAO) Start(ctx context.Context) error {
 	}
 
 	// set init height value
-	if err := dao.kvstore.PutIfNotExists(blockNS, topHeightKey, make([]byte, 8)); err != nil {
-		// ok on none-fresh db
-		if err == db.ErrAlreadyExist {
-			return nil
+	if _, err := dao.kvstore.Get(blockNS, topHeightKey); err != nil {
```

Comment:
does the error type matter? we supposed to put the value if the height doesn't exist